src/os/bluestore/NVMEDevice.cc

```diff
@@ -605,9 +614,12 @@ int NVMEManager::try_get(const spdk_nvme_transport_id& trid, SharedDriverData **
             probe_queue_cond.wait(l);
           }
         }
+        for (auto p : probe_queue) {
+          p->done = true;
+          probe_queue_cond.notify_all();
+        }
```

Comment:
@tone-zhang sorry, i missed this. you might need to do the `notify_all()` out of the `for (auto p : probe_queue)` loop.

Reply:
@tchaikov thanks!